docs/integration.rst

```diff
@@ -447,6 +447,37 @@ BigQueryHook
 .. autoclass:: airflow.contrib.hooks.bigquery_hook.BigQueryHook
     :members:
 
+Compute Engine
```

Comment:
From what I understand those should be in alphabetical order (I saw that in the Guide). So we should put after Cloud Functions :)

Reply:
Let's wait with that for the result of the discussion we have now in GCP naming convention https://docs.google.com/a/polidea.com/document/d/1_rTdJSLCt0eyrAylmmgYc3yZr-_h51fVlnvMmWqhCkY/edit?disco=AAAACJFF76M  - we might want to change the name of the operators anyway as result of this discussion

Reply:
For now let's leave it as it is.